code/modules/antagonists/morph/morph_antag.dm

```diff
@@ -2,5 +2,21 @@
 	name = "Morph"
 	show_name_in_check_antagonists = TRUE
 	show_in_antagpanel = FALSE
+	show_to_ghosts = TRUE
 
-//It does nothing! (Besides tracking)
\ No newline at end of file
+/datum/antagonist/morph/proc/forge_objectives()
+	var/datum/objective/new_objective = new
+	new_objective.owner = owner
+	objectives += new_objective
+	new_objective.explanation_text = "Consume everything." // you can do whatever
```

Comment:
should probably make the objective auto-complete if there's no check for completing it